nttt/cleanup_formatting.py

```diff
@@ -0,0 +1,23 @@
+import re
+from .nttt_logging import log_replacement
+
+
+def trim_formatting_tags(md_file_content, logging):
+    return re.sub(r'(?P<last_word>\S+?)\s*{\s*:\s*(?P<tag>[\w]+?)\s*=\s*"\s*(?P<value>.+?)\s*"\s*}',
```

Comment:
See also the remark for test_cleanup_formatting.py. This regular expression doesn't fix the case {target="_ blank"}. This case occurs in the wild for example in project Boat race, language es-LA, file step_1.md.